src/Algebra/Morphism/TwoMagmas.agda

```diff
@@ -0,0 +1,52 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some properties of Magma homomorphisms
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+
+open import Algebra using (Magma)
+import Algebra.Morphism.Definitions as Definitions
+open import Data.Product using (_,_)
+open import Function using (id; _∘_)
+open import Function.Definitions using (Inverseᵇ)
+open import Relation.Binary using (_Preserves_⟶_)
+import Relation.Binary.Reasoning.Setoid as EqR
+open import Relation.Unary using (Pred)
+
+
+module Algebra.Morphism.TwoMagmas {α α= β β=} (M : Magma α α=) (M' : Magma β β=)
```

Comment:
Please could you rename this module to `Algebra.Morphism.Consequences`

Reply:
Done.
But the name ``Morphism.TwoMagmas`` looks much more informative than                        
``Morphism.Consequences``. The reader would not guess what kind of                          
consequences this module keeps in it.

Reply:
This is the standard name for such modules in the library, e.g. `Relation.Binary.Conquences`, `Relation.Algebra.Consequences`. Better to be consistent.

Reply:
1) My last commit does not contain  ... TwoMagmas. 
2) Instead it has Algebra.Morphism.Consequences, and this was your suggestion.
3) And now you suggest Relation.Algebra.Consequences. 
There is kind of a confusion.
Need I now to move Algebra.Morphism.Consequences to Relation.Algebra.Consequences ?

Reply:
No I mistyped. I was trying to give examples: `Relation.Binary.Conquences` and `Algebra.Consequences`. The file should stay named `Algebra.Morphism.Consequences`.